src/entity/PasswordReset.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseEntity, Column, Entity, JoinColumn, OneToOne, PrimaryColumn } from 'typeorm'
+import { hash } from 'bcrypt'
+import { User } from '../entity/User'
```

Comment:
Import can be shortened:
```suggestion
import { User } from './User'
```

Reply:
Fixed